apps/web/build/fetchFavicon.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import fs from 'node:fs';
+import path from 'node:path';
+
+interface ConfigItem {
```

Comment:
To Do: Create and import both interfaces from an interface file. You can create them in this folder. 

config-item.interface.ts
config-response.interface.ts